UnitTests/GitCommandsTests/GitModuleTests.cs

```diff
@@ -316,6 +316,15 @@ public void ParseRefs()
             Assert.AreSame(_gitModule, refs[3].Module);
         }
 
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase("\t")]
+        [TestCase("012345678abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789")]
```

Comment:
Perhaps a little better...

```suggestion
        [TestCase("A very long string exceeding the limit of length handled (260 characters) 012345678abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqrstuvwxyz01234567")]
```

Reply:
Apologies, I got distracted and didn't see your suggestion until after the merge.
I will fix it up. Thank you